test/upload-blob.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# ./upload-test-blob.sh <EXAMPLE_TENANT_SA_MAIL> <TENANT_ID> <CLIENT_ID> <SUBSCRIPTION_ID> <RESOURCE_GROUP> <STORAGE_ACCOUNT> <CONTAINER_NAME>
+
+EXAMPLE_TENANT_SA_MAIL=$1
+TENANT_ID=$2
+CLIENT_ID=$3
+SUBSCRIPTION_ID=$4
+RESOURCE_GROUP=$5
+STORAGE_ACCOUNT=$6
+CONTAINER_NAME=$7
+
+# get federated token
+GCP_TOKEN=`gcloud auth print-identity-token --impersonate-service-account=${EXAMPLE_TENANT_SA_MAIL} --audiences=api://AzureADTokenExchange`
+echo ${GCP_TOKEN}
```

Comment:
Not sure if we want to `echo` this (and SAS token too) here by default

FYI this is in case you want to consider [print them](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-debug-message) in the future GH action